tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,249 @@
+	})
+
+	var _ = Describe("Creation of the 'Happy path for defult pipeline bundle' resources", func() {
+
+		It("Create buildPipelineDeafultBundle", func() {
+			_, err := framework.CommonController.CreateConfigMap(cm, devNamespace)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Create ReleasePlan in dev namespace", func() {
+			_, err := framework.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, managedNamespace, "")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Create Release Strategy", func() {
+			_, err := framework.ReleaseController.CreateReleaseStrategy(releaseStrategyDefaultName, managedNamespace, releasePipelineNameDefault, releasePipelineBundleDefault, releaseStrategyPolicyDefault, releaseStrategyServiceAccountDefault, paramsReleaseStrategyDefault)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Create ReleasePlanAdmission in managed namespace", func() {
+			_, err := framework.ReleaseController.CreateReleasePlanAdmission(targetReleasePlanAdmissionName, devNamespace, applicationNameDefault, managedNamespace, "", "", releaseStrategyDefaultName) //releaseEnvironment
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates EnterpriseContractPolicy in managed namespace.", func(ctx SpecContext) {
+			_, err := framework.TektonController.CreateEnterpriseContractPolicy(releaseStrategyPolicyDefault, managedNamespace, defaultEcPolicy)
+			Expect(err).NotTo(HaveOccurred())
+		}, SpecTimeout(EnterpriseContractPolicyTimeout))
+
+		It("Create PVC", func() {
+			_, err := framework.TektonController.CreatePVCAccessMode(releasePvcName, managedNamespace, corev1.ReadWriteOnce)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Create Service Account", func() {
+			_, err := framework.CommonController.CreateServiceAccount(releaseStrategyServiceAccountDefault, managedNamespace, managednamespaceSecret)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Create Role", func() {
+			_, err := framework.CommonController.CreateRole(roleName, managedNamespace, roleRules)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Create RoleBinding", func() {
+			_, err := framework.CommonController.CreateRoleBinding(roleBindingName, managedNamespace, subjectKind, roleName, roleRefKind, roleRefName, roleRefApiGroup)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("CreateApplication", func() {
+			_, err := framework.HasController.CreateHasApplication(applicationNameDefault, devNamespace)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("CreateComponent", func() {
+			_, err := framework.HasController.CreateComponent(applicationNameDefault, componentName, devNamespace, gitSourceComponentUrl, "", containerImageUrl, "", "", false)
+			Expect(err).NotTo(HaveOccurred())
+		})
+	})
+
+	var _ = Describe("Post-release verification", func() {
+
+		It("verifies that in dev namespace will be created a PipelineRun.", func() {
```

Comment:
What are we checking a pipelinerun in the dev namespace for?

Reply:
We are talking about e2e test, for a user prescriptive he creates CRDs and application and component 
according to steps, then the validations are what he should see according to the design.

Reply:
I agree that for e2e this is useful, but AIUI this is a test of the build stuff working, not release, so I was surprised to see it in the release suite. The release service doesn't do anything to affect this part

Reply:
You are right in term of we want test only the Release, 
but I think since we are testing a scenario while more than one service is involved, 
what do you say about this  issue @davidmogar @scoheb

Reply:
If it helps debug and trace why the test did not pass...

> i.e. the dev build did not complete therefore the release did not start because of it.

then I would vote to leave it in.

Reply:
I agree with Johnny on this. I wouldn't expect build tests in a release suite. I think you can keep it but i would move it into BeforeAll container and do Expect assertions in the setup container.

Reply:
Thanks @Dannyb48 and @johnbieren 
for your comments,  
so thinking test after your input it does make sense that the creation of CRDs and PR in Dev namespace not be a tests as including them into BeforeAll much reasonable, yet I'll keep it like this now since it's working and will open a separate ticket to fix it as agreed, is it ok by you ?

Reply:
Not sure why it couldn't be done now. If it's working it should just be a matter of restructuring things and moving things around, no?